lib/posix-procfs/README

```diff
@@ -0,0 +1,36 @@
+posix-procfs: implementation of the proc fs
+
+Currently the library offers support for 5 files:
+/proc/meminfo
+/proc/version
+/prov/cmdline
+/proc/filesystems
+/proc/mounts
```

Comment:
This might be better as a table, e.g.:

| Path | Description |
|-|-|
| `/proc/meminfo` | Returns the current memory usage. |

Perhaps (since proc is not really standardized, even the table could suggest which other kernels it supports (e.g. whether the path is linux-specific, *nix, *bsd etc.)